rpc_jobs/rpc_openstack.yml

```diff
@@ -109,9 +109,12 @@
       - "swift"
     action:
       - deploy:
-          FLAVOR: "performance2-15"
+          FLAVOR: "7"
```

Comment:
Hey @hughsaunders, why is the flavor changing here?

Reply:
Its the only flavor thats large enough and doesn't have ephemeral storage. If ephemeral storage is present, the containers are placed there, and not included in the snapshot.

Reply:
Building a quick rpc-openstack build [here](https://rpc.jenkins.cit.rackspace.net/job/PM_rpc-openstack-newton-xenial-swift-deploy/76/console) with a flavor 7 to see how much slower it is to complete on non-SSD.

Reply:
It only took ~ 15 mins longer, so nothing concerning.